website/docs/components/button/partials/guidelines/guidelines.md

```diff
@@ -1,104 +1,78 @@
-## When to use
+## Usage
+
+### When to use
 
 - To trigger an action or event, like a form submission.
 
-## When not to use
+### When not to use
 
 - When navigating to another destination, use an [Inline](/components/link/inline/overview) or [Standalone](/components/link/standalone/overview) link.
 - To display multiple actions under a single button, use [Dropdown](/components/dropdown/overview).
 
----
-
-## Anatomy
-
-![Button anatomy](/assets/components/button/button-anatomy.png)
-
-#### Text\*
-
-Optional (see accessibility)
-
-#### Container
-
-Required
+### Sizes
 
-#### Leading Icon\*
+<Hds::Button @size="small" @text="Small" />
 
-Optional
+<Hds::Button @size="medium" @text="Medium" />
 
-#### Focus Ring
+<Hds::Button @size="large" @text="Large" />
 
-Focus state only
+<Hds::Button @size="large" @isFullWidth={{true}} @text="Full-width button" />
 
-#### Trailing Icon\*
+!!! Insight
 
-Optional
+**Migrating Tip** 
 
-_\* One of these must be present_
+Medium buttons are the same size as default Structure buttons. Small buttons are the same size as compact Structure buttons.
 
----
+!!!
 
-## Size
+Medium is the preferred size, but use a button size that best fits the UI (i.e., don't use a `large` button in a table).
 
-<section>
-  <Hds::Button @size="small" @text="Small" />
-  <Hds::Button @size="medium" @text="Medium" />
-  <Hds::Button @size="large" @text="Large" />
-</section>
-
-Medium is the preferred size, but use a button size that best fits the UI.
-
-For example:
-
-- don't use large buttons in tables.
-
-_Banner (informational):_ Medium buttons are the same size as default Structure buttons. Small buttons are the same size as Compact Structure buttons.
+Buttons are able to be manually stretched to accommodate a need for a full-width button. Full-width buttons should fill 100% of the parent container.
 
-<section>
-  <Hds::Button @size="large" @isFullWidth={{true}} @text="Full-width button" />
-</section>
+!!! Info
 
-Buttons are able to be manually stretched to accommodate a need for a full-width button. Full-width buttons should fill 100% of the parent container.
+**Figma Tip:** Select the `button` layer and manually adjust the width to the width needed, then select the child layer (i.e., `\_main/[size]`) and change _Resizing_ for width to 'Fill Container'
 
-_Banner (informational):_ **Figma Tip:** Select the 'button' layer and manually adjust the width to the wdith needed, then select the child layer (ie '\_main/[size]') and change _Resizing_ for width to 'Fill Container'
+!!!
```

Comment:
It's ok to remove these if you see them, as our Figma Tips page will be staying directly in Figma.